addon/components/simple-chart.js

```diff
@@ -34,6 +34,14 @@ export default Component.extend({
         this.set('tooltipLocation', tooltipLocation);
       }
     },
+    click(data, slice, tooltipLocation){
```

Comment:
I did work with one on another app in the past. (see screenshot)
<img width="382" alt="screen shot 2017-06-29 at 4 44 29 pm" src="https://user-images.githubusercontent.com/597090/27715191-50251f22-5cea-11e7-94cf-4bce0c2ec9c5.png">
 I can remove it and you can always add it back later if you ever find a need for one.